.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,101 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0,3' # 2/weekly
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
+        platform: [x64]
+        PLAT: [x86_64]
+    env:
+      PKG_NAME: gensim
+      BUILD_COMMIT: 4.0.0beta
```

Comment:
I think it should build the latest HEAD here. There's no point building the same commit every two weeks, is there?